assembl/lib/frontend_urls.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, discussion):
 
     @classmethod
     def register_frontend_routes(cls, config):
-        from assembl.views.backbone.views import home_view
+        from assembl.views.discussion.views import home_view
```

Comment:
I think so too. this module contains both backbone and react views, hence the rename. (plus, backbone vs home was confusing)